oshi-core/src/main/java/oshi/hardware/common/AbstractPowerSource.java

```diff
@@ -71,4 +71,10 @@ public double getRemainingCapacity() {
     public double getTimeRemaining() {
         return this.timeRemaining;
     }
+
+    /** {@inheritDoc} */
+    @Override
+    public String toString() {
+        return String.format("%n %s @ %.1f%%", this.getName(), this.getRemainingCapacity() * 100d);
```

Comment:
Let's not include the `%n ` header here.

Reply:
Also while it wasn't in the original `SystemInfoTest` we should add `getTimeRemaining()`.  If positive, seconds remaining (would be awesome to format as something like `1h 20m 15s` or even omit seconds and do `1:20`). If negative, -1.0 (calculating) or -2.0 (unlimited)... careful with floating point equality here too, you probably want to do "<1.5" and "<0" cases.